src/main/resources/application.yaml

```diff
@@ -1,13 +1,20 @@
 db:
   name: user_management
   user: postgres
-  password: root
```

Comment:
La fisierul de proprietati sa incercam sa nu ii facem push decat in cazul in care ticketul necesita modificari in acest fisier